src/scripts.js

```diff
@@ -398,7 +398,7 @@ function showAllRecipes() {
     domRecipe.style.display = 'block'
   })
 
-  main.classList.remove('display-favorites')
+  recipeContainer.classList.remove('display-favorites')
```

Comment:
This line (and I think another one) needs to add the `container` class back to this line. Otherwise, once you go to the favorites page, `main` will stick with `display: block;`

we can definitely fix this later though